scootapi/setup/workers.go

```diff
@@ -118,6 +118,9 @@ func NewGitDB(tmpDir *temp.TempDir, repoDir, storeAddr string) (snapshot.DB, err
 		}
 	} else {
 		r, err = repo.NewRepository(repoDir)
+		if err != nil {
```

Comment:
Yes, it probably should, but I really don't want to pull on that string in this PR.

As to style, golang goes the other way:
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow